lib/league_stats.rb

```diff
@@ -67,7 +71,7 @@ def highest_scoring_visitor
       sum_goals = games.sum do |game|
         game.goals.to_f
       end
-      (sum_goals / games.length).round(2)
```

Comment:
This implementation of our average mathable module works great across multiple classes.